extensions/applicationinsights-properties-js/src/Context/User.ts

```diff
@@ -176,4 +196,8 @@ export class User implements IUserContext {
     public clearAuthenticatedUserContext() {
         // @DynamicProtoStub -- DO NOT add any code as this will be removed during packaging
     }
+
+    public update() {
```

Comment:
While this does get removed after converted to JS, the function signature needs to match as this is used by TypeScript to generate the *.d.ts files.

Reply:
ah sorry that was a typo. Added it, thanks.